### PR TITLE
implements generators of fake data (contacts, notes)

### LINKED ISSUE #12: Fill initial data with mocker
Add a command "fake-users".

### PR DESCRIPTION
**Pull request by UraFIZ**
closed #12

### CODE REVIEW

File: src/faker_generator.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from src.constants import INFO
+
+fake = Faker()
+
+def generate_tags():
+    possible_tags = ["work", "personal", "urgent", "important", "low-priority", "ideas", "to-do"]
+    num_tags = random.randint(1, len(possible_tags))
+    return random.sample(possible_tags, num_tags)
+
+
```
Comment (nrkshaman) [inline src/faker_generator.py:18]: add @input_error decorator to handle errors in args parsing, also add str to int casting error in decorator if needed

File: src/faker_generator.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        contact = Record(name)
+        contact.add_phone(address_book[name]["phones"])
+        contact.add_email(address_book[name]["email"])
+        contact.add_address(address_book[name]["address"])
+        contact.add_birthday(address_book[name]["birthday"])
+
+        book.add(contact)
+
+    return INFO + f" {number} contacts successfully generated"
+
```
Comment (nrkshaman) [inline src/faker_generator.py:41]: @input_error

**Review by nrkshaman — CHANGES_REQUESTED** [review]

File: src/faker_generator.py (hunk 1 above)
Comment (UraFIZ) [reply src/faker_generator.py]: thanks! done

File: src/faker_generator.py (hunk 2 above)
Comment (UraFIZ) [reply src/faker_generator.py:42]: thanks! done

**Review by nrkshaman — APPROVED** [review]

**Pull request merged**